logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/e2ei/usecase/EnrollE2EIUseCase.kt

```diff
@@ -65,7 +72,31 @@ class EnrollE2EIUseCaseImpl internal constructor(
             return E2EIEnrollmentResult.Failed(E2EIEnrollmentResult.E2EIStep.AcmeNewAuthz, it).toEitherLeft()
         }, { it })
```

Comment:
BTW, `mapLeft {... }` instead of `fold({...}) { it }`might make code-style much better imo 
As far as you already edited this file :)

Reply:
Thanks buddy, good catch. since I needed that 'it' to log details, forgot to make it better. thanks again for the great tip 💪